src/main/java/seedu/address/logic/parser/TimeParserUtil.java

```diff
@@ -24,8 +24,8 @@
             + "An example of a properly formatted time includes (but is not limited to): 2 PM\n"
             + "An example of a natural time language is 'morning' (8am) or 'evening' (7pm)"
             + "Please visit our user guide at "
```

Comment:
I think can remove `at`?
Just like the one in DateParserUtil.java

Reply:
Thanks for spotting this! Missed this out when I was editing it 😄